api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -545,14 +562,15 @@ ml_single_invoke (ml_single_h single,
     }
   }
 
-  if (single_h->data_ready == TRUE) {
+  if (single_h->state == RUNNING) {
```

Comment:
It seems that `single_h->state` should be `IDLE` to work. How about merging these state checking into a single line like `if (single_h->state != IDLE)` at the entry?

Reply:
You are right, updated.